tests-legacy/Integration/PrestaShopBundle/Controller/Api/Improve/Design/PositionsControllerTest.php

```diff
@@ -52,6 +53,12 @@ protected function setUp()
         /** @var ModuleManager */
         $moduleManager = self::$kernel->getContainer()->get('prestashop.module.manager');
 
+        // Unregister all modules hooked on displayHome
+        Db::getInstance()->execute('
+            DELETE FROM `' . _DB_PREFIX_ . 'hook_module`
+            WHERE `id_hook` = '. (int) Hook::getIdByName('displayHome')
+        );
```

Comment:
This test wasn't resilient enough if an unexpected module was installed too, which was actually happening with some locales. From now, we build the proper environment for this test, which is having ONLY 2 modules, and all the other one being removed from the hook list.